tests/src/test/java/alluxio/client/FileSystemIntegrationTest.java

```diff
@@ -79,7 +79,7 @@ public void createFileWithFileAlreadyExistsException() throws Exception {
     mFileSystem.createFile(uri, mWriteBoth).close();
     Assert.assertNotNull(mFileSystem.getStatus(uri));
     try {
-      mFileSystem.createFile(uri, mWriteBoth);
+      mFileSystem.createFile(uri, mWriteBoth).close();
```

Comment:
Is there a way to ensure all clients are cleaned up after tests? It seems difficult to try and enforce good code practice everywhere. For example here if createFile fails, there is a chance we will not close the client right (though you can argue `createFile` should ensure all resources are cleaned up on a failure)?